### PR TITLE
Add japan building data sandcastle

### PR DESCRIPTION
**Pull request by jjspace**
# Description

- create a sandcastle to highlight our new Japan building dataset
- Includes a few POIs in different locations to showcase the data
    - I planned to add translations for the dropdown, those should come shortly.
- I opted for a light that comes from the camera to avoid nighttime making the buildings darker
- I turned on antialiasing as it helps a lot with the edges of buildings especially in the areas that have actual imagery textures. @ggetz is there anything else that might make sense as "easy wins" to increase the visual quality?

## Issue number and link

No issue

## Testing plan

- Run `npm start`
- open sandcastle and navigate to the [Japan Buildings one](http://localhost:8080/Apps/Sandcastle/index.html?src=Japan%20Buildings.html)
- Test the sandcastle and make sure it behaves well
- Make sure it shows up in search and the categories make sense
- Make sure the thumbnail looks good

# Author checklist

- [ ] I have submitted a Contributor License Agreement
- [ ] I have added my name to `CONTRIBUTORS.md`
- [ ] I have updated `CHANGES.md` with a short summary of my change
- [ ] I have added or updated unit tests to ensure consistent code coverage
- [ ] I have update the inline documentation, and included code examples where relevant
- [ ] I have performed a self-review of my code

### CODE REVIEW

File: Apps/Sandcastle/gallery/Japan Buildings.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+              -0.026110036454204615,
+              0.335331963065526
+            ),
+            up: new Cesium.Cartesian3(
+              -0.2518896785254185,
+              0.6059364940549604,
+              0.7545810460280222
+            ),
+          },
+          duration: 0,
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Japan Buildings.html:121]: Why is this the only POI with `0` duration?

File: Apps/Sandcastle/gallery/Japan Buildings.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+          easingFunction: Cesium.EasingFunction.LINEAR_NONE,
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Japan Buildings.html:122]: Is this necessary with the duration of `0`?

Comment (jjspace) [reply Apps/Sandcastle/gallery/Japan Buildings.html:122]: no need

**Review by jjhembd — APPROVED** [review]
Looks good to me, thanks @jjspace !

**Pull request merged**